### ISSUE TITLE
Possible missing dependency on babel-runtime

### ISSUE THREAD

**Issue by christopherthielen** [webpack-contrib/karma-webpack#307]
**I'm submitting a bug report**

**Webpack version:**
3.11.0

**Webpack Karma version**:
1.7.x

**Karma version**:
2.0.11

**Please tell us about your environment:**
OSX

**Current behavior:**

```
$ karma start
25 02 2018 19:56:08.367:ERROR [config]: Error in config file!
 { Error: Cannot find module 'babel-runtime/helpers/toConsumableArray'
    at Function.Module._resolveFilename (module.js:536:15)
    at Function.Module._load (module.js:466:25)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/cthielen/projects/uirouter/sticky-states/node_modules/karma-webpack/lib/karma-webpack.js:3:27)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32)
    at tryModuleLoad (module.js:497:12)
    at Function.Module._load (module.js:489:3)
    at Module.require (module.js:579:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/cthielen/projects/uirouter/sticky-states/node_modules/karma-webpack/lib/index.js:7:18)
    at Module._compile (module.js:635:30)
    at Object.Module._extensions..js (module.js:646:10)
    at Module.load (module.js:554:32) code: 'MODULE_NOT_FOUND' }
```

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem along with a gist/jsbin of your webpack configuration.**

```
git clone https://github.com/ui-router/sticky-states.git
cd sticky-states
git checkout karma-webpack
yarn
yarn add --dev karma-webpack@2.0.11
yarn test
```

Adding `babel-runtime` as a dep fixes the issue.  Guessing this was due to https://github.com/webpack-contrib/karma-webpack/commit/6d28c5cf1246e1659a7553c18af6f43de84a7db5.  2.0.9 was the last version of karma-webpack that worked without babel-runtime.

**Comment by dmk1111** [comment]
We had to lock version to 2.0.9 as this breaking change was not announced properly

**Comment by michael-ciniawsky** [comment]
Could someone please quick review (#310)

**Comment by michael-ciniawsky** [comment]
Released in [`v2.0.12`](https://github.com/webpack-contrib/karma-webpack/releases/tag/v2.0.12) 🎉

**Comment by CaseyPeters** [comment]
@michael-ciniawsky This is still failing for me in `v2.0.12`

**Comment by michael-ciniawsky** [comment]
😞 Could include some context (`{Error}` Logs,` package.json` etc), did clean the `npm/yarn cache` and updated the `lockfil` etc ?

**Comment by CaseyPeters** [comment]
@michael-ciniawsky updated my previous comment with out console output. We are using it as a dev-dependency so it will not be included in our lockfile. I did clear the `npm cache`

**Comment by arturs-razmuss** [comment]
On project with 
webpack@3.7.1
karma@1.7.1
karma-webpack@2.0.12
the following error is encountered. 
```
ERROR [plugin]: Error during loading "C:\project\node_modules/karma-webpack" plugin:
  Cannot find module 'babel-runtime/helpers/toConsumableArray'
ERROR [preprocess]: Can not load "webpack", it is not registered!
```
This seems to be caused by the fact project doesn't use babel and babel-runtime is not provided as dependency by karma-webpack. Thus require call fails on _\node_modules\karma-webpack\lib\karma-webpack.js:3_
```
var _toConsumableArray2 = require('babel-runtime/helpers/toConsumableArray');
```

**Comment by rossgardt** [comment]
Not sure why there is no notice, but just FYI: this was fixed in #312.
